language/bytecode_verifier/bytecode_verifier_tests/src/unit_tests/signature_tests.rs

```diff
@@ -8,7 +8,248 @@ use invalid_mutations::signature::{
 };
 use proptest::{collection::vec, prelude::*};
 use types::vm_error::StatusCode;
-use vm::file_format::CompiledModule;
+use vm::file_format::*;
+
+fn test_module() -> CompiledModuleMut {
```

Comment:
I copied this function from my other PR. I'll make sure we reuse it once we check it in.